appveyor.yml

```diff
@@ -48,8 +48,8 @@ after_build:
 - 7z e DiscImageCreator_20230606.zip -oMPF\bin\Debug\net8.0-windows\win-x64\publish\Programs\Creator Release_ANSI\*
 
 # Redumper
-- ps: appveyor DownloadFile https://github.com/superg/redumper/releases/download/build_294/redumper-2023.12.08_build294-win64.zip
-- 7z e redumper-2023.12.08_build294-win64.zip -oMPF\bin\Debug\net8.0-windows\win-x64\publish\Programs\Redumper redumper-2023.12.08_build294-win64\bin\*
+- ps: appveyor DownloadFile https://github.com/superg/redumper/releases/download/build_306/redumper-2023.12.29_build306-win64.zip
+- 7z e redumper-2023.12.29_build306-win64.zip -oMPF\bin\Debug\net8.0-windows\win-x64\publish\Programs\Redumper redumper-2023.12.29_build306-win64\bin\*
```

Comment:
I would prefer if you do this in a separate change. If you want to add it here, please add support for any new or changed parameters. I think I saw `dumpnew` being the only one this time around.